README.rst

```diff
@@ -11,32 +11,30 @@ Installation
 
 ::
 
-    $ pipenv install geomac-wildfires
+    $ pipenv install nifc-wildfires
 
 
 Command-line usage
 ------------------
 
 ::
 
-    Usage: geomacwildfires [OPTIONS] COMMAND [ARGS]...
+    Usage: nifcwildfires [OPTIONS] COMMAND [ARGS]...
 
       A command-line interface for downloading wildfire perimeter data from
-      GeoMAC. Returns GeoJSON.
+      NIFC. Returns GeoJSON.
 
     Options:
       --help  Show this message and exit.
 
     Commands:
-      active-perimeters   Perimeters of active fires in a recent 24-hour period
-      all-perimeters      Perimeters of all fires
+      current-perimeters   Perimeters of active fires
```

Comment:
Is there a difference between being current and being active?

Reply:
No. which is why we are only keeping one - "current-perimeters"

Reply:
Gotcha. Does current mean the same thing as active? My only little picky thought is that we should try to use the same term across all of our repositories. For instance, if we use current here, we shouldn't use active in another one.

Reply:
ok I will stick with "Active"